Makefile

```diff
@@ -88,3 +88,8 @@ update_version:
 
 	grep "$(PLACEHOLDER)" "setup.py"
 	sed -i "s/$(PLACEHOLDER)/__version__ = \"${VERSION}\"/g" "setup.py"
+
+# TODO
+.PHONY: clear-cache
+clear-cache:
```

Comment:
can we make this a pyflyte command?

`pyflyte cache clear`

Reply:
good call, much cleaner.